build/configs/rtl8721csm/loadable_apps/defconfig

```diff
@@ -102,6 +103,7 @@ CONFIG_REG_STACK_OVERFLOW_PROTECTION=y
 #
 CONFIG_ARMV8M_HAVE_ICACHE=y
 CONFIG_ARMV8M_HAVE_DCACHE=y
+# CONFIG_ARMV8M_LAZYFPU is not set
```

Comment:
You said that enabling FPU, the handling of interrupts is faster.
Why don't we enable then?
What is the trade-off?

Reply:
FPU is not enabled in RTK. So, we dont need to use LAZYFPU setting. 
Also, the drawback of LAZYFPU is that we cannot use floating point operations in interrupt handler because previous FPU state is not saved.